File: front/src/pages/interface/Category.ts
export enum Category {
  SPORT = "SPORT",
  COUNTRIES = "COUNTRIES",
  ENTERTAINMENT = "ENTERTAINMENT",
  HISTORY = "HISTORY",
  SCIENCE = "SCIENCE",
  GEOGRAPHY = "GEOGRAPHY"
}
export const CategoryDisplayNames: { [key in Category]: string } = {
    [Category.SPORT]: "ספורט",
    [Category.COUNTRIES]: "מדינות",
    [Category.ENTERTAINMENT]: "בידור",
    [Category.HISTORY]: "היסטוריה",
    [Category.SCIENCE]: "מדע",
    [Category.GEOGRAPHY]: "גאוגרפיה",
  };
  

File: backend/src/main/java/com/trivia/trivia/game/Entity/Category.java
package com.trivia.trivia.game.Entity;


public enum Category {
    SPORT("ספורט"),
    COUNTRIES("מדינות"),
    ENTERTAINMENT("בידור"),
    HISTORY("היסטוריה"),
    SCIENCE("מדע"),
    GEOGRAPHY ("גאוגרפיה");

    private final String displayName;

    Category(String displayName) {
        this.displayName = displayName;
    }

    public String getDisplayName() {
        return displayName;
    }
}
